settings.gradle

```diff
@@ -4,4 +4,5 @@ include 'email-client'
 include 'domain-model'
 include 'domain-sample-data'
 include 'ccd-adapter'
+include 'ccd-client'
```

Comment:
The client should be moved to its own repository, published to jcenter, made reusable for everyone. No point developing another copy of CCD client in private.

Reply:
I also have this in mind, but we have to move `ccd-mapper` component first and for that we need to move `domain-model` as well. Probably a separate PR to do this refactoring.

Reply:
why does ccd-mapper need to be extracted @sabahirfan ?

Reply:
as `ccd-adapter` is dependency for this module.

Reply:
why?

Reply:
`CCDCase` from `ccd-adapter` is required by this service client.